client/webserver/site/src/html/markets.tmpl

```diff
@@ -423,14 +423,15 @@
           </div>
         </div>
         <hr class="dashed my-3">
-        <div class="fs16 text-center">Authorize this order with your app password.</div>
-        <div class="d-flex mt-3">
-          <div class="col-12 p-0 text-left">
-            <label for="vPass" class="pl-1 mb-1">Password</label>
-            <input type="password" class="form-control select" id="vPass">
-          </div>
-          <div class="col-12 p-0 text-right">
-            <div>&nbsp;</div>
+        {{$passwordIsCached := .UserInfo.PasswordIsCached}}
+        <div class="fs16 text-center {{if $passwordIsCached}}d-hide{{end}}">Authorize this order with your app password.</div>
+        <div class="d-flex mt-3 {{if $passwordIsCached}}justify-content-center{{end}}" >
+            <div class="col-12 p-0 text-left {{if $passwordIsCached}}d-hide{{end}}">
+              <label for="vPass" class="pl-1 mb-1">Password</label>
+              <input type="password" class="form-control select" id="vPass">
+            </div>
+          <div class="col-12 p-0 {{if not $passwordIsCached}}text-right{{end}}">
```

Comment:
Indentation is not consistent. Might need to change some settings in your IDE or something so this doesn't happen.